app/src/parser_impl.c

```diff
@@ -384,14 +465,14 @@ parser_error_t parse_version(parser_context_t *ctx, uint32_t *version){
 parser_error_t check_entrypoint(parser_context_t *ctx, ExecutableDeployItem *item, uint32_t *num_runs){
     char buffer[100];
     MEMZERO(buffer, sizeof(buffer));
+    uint16_t offset = ctx->offset;
```

Comment:
we could update entry_point_offset here and avoid having temporal offset variable